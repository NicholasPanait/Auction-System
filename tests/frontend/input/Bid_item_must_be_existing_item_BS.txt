login
testBSUser
password
bid
itemDoesNotExist
fakeSeller
logout
