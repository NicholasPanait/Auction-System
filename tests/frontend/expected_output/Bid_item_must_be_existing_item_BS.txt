Welcome to the Online Auction Sales Service!

Please enter login to Login.
login - Login

***LOGIN***
Enter Username: 
Enter Password: 
Welcome Buy-Standard User: testBSUser!

***MENU***
logout - Logout
bid - Make a bid on an available item
addcredit - Add Credit
listitems - List Items Up For Auction
changepassword - Change Your Password
***MAKING A BID***

Enter Item Name: 
Enter seller's username: Item sold by this seller does not exist

***MENU***
logout - Logout
bid - Make a bid on an available item
addcredit - Add Credit
listitems - List Items Up For Auction
changepassword - Change Your Password

You have successfully logged out.
